plugins/music.js

```diff
@@ -5,11 +5,23 @@ module.exports = function(pluto) {
 
     var musicModule = {
         lastPlaying: null,
+        progress: null,
         queue: [],
         paused: false,
         lastMessage: null
     };
 
+    pluto.addListener("player::progress", function(data) {
```

Comment:
i think websockets might be overkill for this feature, i think just make polling less sketchy